src/createClient.js

```diff
@@ -43,7 +43,7 @@ export default function createClient(address, options = {}) {
    * @param {Function} callback A callback that is called when response has been received
    * or request failed.
    */
-  const send = (method, params, callback) => {
+  const send = (method, params, callback, timeout=clientOptions.timeout) => {
```

Comment:
Callback must be the last argument, i would rather expose the options to do quick manual change or create `setOptions({` method.